gdk-pixbuf/src/auto/pixbuf_format.rs

```diff
@@ -29,9 +29,9 @@ impl PixbufFormat {
 
     #[doc(alias = "gdk_pixbuf_format_get_extensions")]
     #[doc(alias = "get_extensions")]
-    pub fn extensions(&self) -> Vec<glib::GString> {
+    pub fn extensions(&self) -> Option<Vec<glib::GString>> {
         unsafe {
-            FromGlibPtrContainer::from_glib_full(ffi::gdk_pixbuf_format_get_extensions(
+            TryFromGlibPtrContainer::try_from_glib_full(ffi::gdk_pixbuf_format_get_extensions(
```

Comment:
I would instead name it, MaybeFromGlibPtrContainer or so. Try as a prefix assumes we would get a Result back instead of an Option

Reply:
I am also not sure if all those returned `Option<Vec<T>>` make sense or some of them needs annotations fixes to return just `Vec<T>` that would be empty if there are no items.

Reply:
> I would instead name it, MaybeFromGlibPtrContainer or so. Try as a prefix assumes we would get a Result back instead of an Option

Indeed. done.

> I am also not sure if all those returned Option<Vec<T>> make sense or some of them needs annotations fixes to return just Vec<T> that would be empty if there are no items.

Yes, I went looking at the documentation for these functions and spotted some discrepancies last time I tackled this. I'll force annotations with my conclusions in a bit. I just wanted to rebase & push now that I have something usable as a whole (last time I paused, I had just figured out I had to implement all those `Maybe` traits).